cdap-ui/app/features/tracker/controllers/integrations-ctrl.js

```diff
@@ -63,16 +83,28 @@ class TrackerIntegrationsController {
 
     this.pollId = null;
 
+    $scope.$watch(() => {
+      return this.navigatorSetup.isOpen;
+    }, () => {
+      if (!this.navigatorSetup.isOpen && this.navigatorSetup.isSetup) {
+        this.navigatorSetup.popoverEnabled = false;
+      }
+    });
+
   }
 
   getKafkaBrokerList() {
     this.myTrackerApi.getCDAPConfig({ scope: this.$scope })
       .$promise
       .then( (res) => {
-        let filtered = res.filter( (config) => {
-          return config.name === 'metadata.updates.kafka.broker.list';
+        let zookeeperQuorum = res.filter( (config) => {
+          return config.name === 'zookeeper.quorum';
+        });
+
+        let zookeeperKafka = res.filter( (config) => {
+          return config.name === 'kafka.zookeeper.namespace';
         });
-        this.navigatorInfo.brokerString = filtered[0].value;
+        this.navigatorInfo.metadataKafkaConfig.zookeeperString = zookeeperQuorum[0].value + '/' + zookeeperKafka[0].value;
```

Comment:
Could there be a possibility that one of the values `zookeeperQuorum or zookeeperKafka` is an empty array?